scheduler/scheduler_sync.daemon

```diff
@@ -79,37 +92,47 @@ endtime=0
 inotifywait --monitor --timefmt '%Y%m%d.%H:%M:%S' --format '%T %w %e %f' -e modify,attrib $WATCH_DIR | \
 while read TIME DIRECTORY ACTION FILE # These must line up with format parameters
 do
-    # The conditional filters the inotify output to only trigger the syncronization when the .scd
+    # The conditional filters the inotify output to only trigger the synchronization when the .scd
     # file for the site specified by RADAR_ID is modified.
     if [[ "$FILE" =~ .*$RADAR_ID\.scd$ ]]
     then
         # Only execute the code once every $filter_time seconds
         cur_time=$(date "+%s")
-        if [[ $(($endtime + $filter_time)) -lt $cur_time ]]; then
-            printf "$(date --utc "+%Y%m%d %H:%M:%S UTC") - Schedule modified for $RADAR_ID\n"
+        if [[ $((endtime + filter_time)) -lt $cur_time ]]; then
+            printf "%s - Schedule modified for %s\n" "$(date --utc "+%Y%m%d %H:%M:%S UTC")" "$RADAR_ID"
             sleep 1     # Wait 1 s to ensure all changes to scd file are finished
 
             # Sync the .scd file to site via Maxwell
-            printf "Sending ${SCD_FILE} to ${DEST}:${DEST_DIR}\n"
+            printf "Sending %s to %s:%s via port %s\n" "${SCD_FILE}" "${SCHEDULER_DEST}" "${DEST_DIR}" "${DEST_PORT}"
             retval=1
             while [[ $retval -ne 0 ]]; do
-                rsync -av --rsh="ssh $MAXWELL ssh -p $DEST_PORT" "${SCD_FILE}" "${DEST}:${DEST_DIR}"
+                rsync -av -p -e "ssh -p $DEST_PORT" "${SCD_FILE}" "${SCHEDULER_DEST}:${DEST_DIR}"
```

Comment:
```suggestion
                rsync --archive --verbose --perms --rsh="ssh -p $DEST_PORT" "${SCD_FILE}" "${SCHEDULER_DEST}:${DEST_DIR}"
```
I like the more descriptive command options - it makes it easier to understand what exactly is happening. This is just preference though

Reply:
That's a good point, I like that better too.